package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-app-scheduler",
-  "version": "5.1.0",
+  "version": "6.0.0",
```

Comment:
Please revert this part of the update. We generally like to bump the version as part of the release process itself.